lib/block.rb

```diff
@@ -0,0 +1,18 @@
+module Hotel
+  class BlockOfRooms
+    attr_reader :room_numbers, :date_range, :room_rate
+
+    MAX_ROOM_BLOCK = 5
+
+    def initilize(room_rate, date_range, room_numbers)
+      unless room_numbers.length <= MAX_ROOM_BLOCK
+        raise ArgumentError
+      end
```

Comment:
You should always include a message when you raise an error. That way whoever messed up and triggered it knows what they did wrong. Whether it's a coworker or you in 6 months, they'll be grateful.